### PR TITLE
ref(js): Move OrganizationContext + RouteContext -> utils

### PR DESCRIPTION
**Pull request by evanpurkhiser**
These contexts are not exclusively used in the views, they don't really
belong in here

### CODE REVIEW

**Comment by priscilawebdev** [comment]
We will have to update the imports in getsentry too

File: static/app/views/dashboards/dashboard.spec.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {MEPSettingProvider} from 'sentry/utils/performance/contexts/metricsEnhan
 import Dashboard from 'sentry/views/dashboards/dashboard';
 import {DisplayType, Widget, WidgetType} from 'sentry/views/dashboards/types';
 
-import {OrganizationContext} from '../organizationContext';
+import {OrganizationContext} from '../../utils/organizationContext';
```
Comment (ryan953) [inline static/app/views/dashboards/dashboard.spec.tsx:9]: Time to swap this to be `from 'sentry/utils/organizationContext';`

File: static/app/views/organizationContextContainer.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ import RequestError from 'sentry/utils/requestError/requestError';
 import withApi from 'sentry/utils/withApi';
 import withOrganizations from 'sentry/utils/withOrganizations';
 
-import {OrganizationContext} from './organizationContext';
+import {OrganizationContext} from '../utils/organizationContext';
```
Comment (ryan953) [inline static/app/views/organizationContextContainer.tsx:29]: Time to swap this to be `from 'sentry/utils/organizationContext';`

**Review by ryan953 — COMMENTED** [review]
Maybe we should leave the file `static/app/views/routeContext.tsx` on the filesystem, but have it import & export everything from `sentry/utils/routeContext`. 

Then the getsentry imports can be changed over, and a 3rd PR can come and remove the leftover views/routeContext file afterwards.

ditto organizationContext.

**Pull request closed without merging**